app/uk/gov/hmrc/entrydeclarationdecision/utils/Timer.scala

```diff
@@ -35,6 +38,15 @@ trait Timer {
     def startTimer(metric: Metric): Timer.Context = metrics.defaultRegistry.timer(s"$metric-timer").time()
   }
 
+  def isLongJourneyTime(startTime: Instant, longJourneyTime: FiniteDuration): Boolean = {
```

Comment:
Does not feel like this belongs here: this trait is a general purpose timing/metrics utility (& shared between projects) but this is logic for a specific business requirement.